cluster/images/hyperkube/master.json

```diff
@@ -37,6 +60,25 @@
               "--master=127.0.0.1:8080",
               "--v=2"
         ]
+    },
+    {
+      "name": "setup",
```

Comment:
Works, doesn't it? :) We can make it cleaner with init containers which might make it into 1.2.
https://github.com/kubernetes/kubernetes/issues/1589